scripts/allowlist/README.md

```diff
@@ -0,0 +1,79 @@
+# Genesis File Content
+
+This directory contains a script (`createContent.js`) that generates some content that needs to be added to the genesis 
+file of a network that makes use of the allowlist smart contracts (`contracts/allowlist`).
+
+### Install the dependencies
+
+To install the dependencies of the script run `npm install` in the `scripts/allowlist` directory. 
+
+### Input File
+
+The script reads the file `allowedAccountsAndValidators.txt` that defines in each line the address of an allowed account and
+(optionally) the validator for that account. The account and the validator are specified using their address as a 
+hexadecimal string. If the account has a nominated validator, the account and the validator hexadecimal strings need
+to be separated by a comma:
+
+    0x5B38Da6a701c568545dCfcB03FcB875f56beddC4, 0xAb8483F64d9C6d1EcF9b849Ae677dD3315835cb2
```

Comment:
Might be more canonical if it would be json input instead of manually creating a parser. But not important.

Reply:
You might be right and I had thought about it, but came to the conclusion that it would be easier to do it this way ...

Reply:
You are right, but I thought it would be easier to create and to program it this way ...